### PR TITLE
fix(curriculum): change text in user story to match test case

### PR DESCRIPTION
**Pull request by mathbraga**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46446

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ dashedName: build-a-tribute-page
 
 **User Stories:**
 
-1. Your tribute page should have an element with a corresponding `id="main"`, which contains all other elements
+1. Your tribute page should have a `main` element with a corresponding `id` of `"main"`, which contains all other elements
```
Comment (jdwilkin4) [inline curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md:15]: I don't think main should be wrapped in quotes here
`id` of `"main"`

If you look at steps 2&3, there are no quotes
`id` of `title`
`id` of `img-div`

Quotes were only used in the context of the attribute 
`id="main"`

Proposed fix: `id` of `main`

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md (hunk 1 above)
Comment (mathbraga) [reply curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md:15]: Makes sense, thank you for the additional idea. I'll do the change right away.

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Review by jeremylt — APPROVED** [review]
LGTM

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**